packages/plugin-linebreak/LineBreakDomParser.ts

```diff
@@ -5,16 +5,18 @@ export class LineBreakDomParser extends AbstractParser<Node> {
     static id = 'dom';
 
     predicate = (item: Node): boolean => {
-        return item.nodeName === 'BR';
+        return item instanceof Element && item.nodeName === 'BR';
     };
 
-    async parse(item: Node): Promise<LineBreakNode[]> {
+    async parse(item: Element): Promise<LineBreakNode[]> {
         if (!item.nextSibling) {
             // A <br/> with no siblings is there only to make its parent visible.
             // Consume it since it was just parsed as its parent element node.
             // TODO: do this less naively to account for formatting space.
             return [];
```

Comment:
You used to handle attributes on the second `BR`. I don't think you have to, but if you want to, you could use the same "hack" as the attributes on `LI`. (With the same shortcomings that it should be removed if the situation around the LineBreak changes.)